### DISCUSSION TITLE
这或是又一个信息茧房

### DISCUSSION THREAD

**Discussion by theCNONE** [WeNeedHome/SummaryOfLoanSuspension#601] [General]
这里面统计的是真正完完全全的断供，还是只是烂尾楼？ 
晚上有很多所谓的“烂尾声明”，当然有真的，那是否有假的呢？反正那些“声明”上既没有盖章，也没手印更没签字 
我现在发布“重庆保利观澜三期全体业主停贷声明”，是仅仅我停贷，还是真的所有人联合停贷呢？ 
所谓的停贷里面又有多少“工贼”呢？ 
在“航空母舰吧”全是兔兔，在“中国房价吧”里面全是神神，在“抗吧”里面全是鼠鼠，真正的因停贷导致的不良资产又占银行资产的多少呢？

**Comment by theCNONE** [comment]
很难不支持

**Comment by SiminePu** [comment]
本来我也想整理的，但是想了想也是，有图也不一定是真的，更何况这个仓库里也没两百张图。

**Reply by theCNONE** [reply]
我看过那些声明，就是某某楼盘全体业主开始停贷，下面全是套话，根本没办法证实

**Comment by 1510207073** [comment]
别的地方我不知道，至少我周围的小区在这上面的都是真实的

**Reply by theCNONE** [reply]
不否认有真的，但也不排除有假的。问题是这金融风险真的大到不可收拾吗？我感觉没雷曼兄弟的大

**Reply by zcaaron** [reply]
那是你的无知了，风险比雷曼大了去了。

**Reply by MrChen2016-08-27** [reply]
雷曼回收房子，还可以降低价格拍卖70%，这个回收楼花只能拍卖空气

**Reply by SiminePu** [reply]
中国无限责任制可不光可以收你房子

**Reply by ttimasdf** [reply]
> 不否认有真的，但也不排除有假的。问题是这金融风险真的大到不可收拾吗？我感觉没雷曼兄弟的大

当然没有。这些声明本身是没啥意义的。银行、金融机构和监管部门说什么和做什么，才是对旁观者最有意义的。银行这两天都在密集的发声明说没有系统性危机，数据有没有水分另说，但是监管部门至少提前几个月就组织过自查，并看过这些真实数据了。

现在新闻所看到的，包括这个repo本身，只是市场情绪的体现而已。经济是靠数据说话的，不是靠情绪。所以说，不要想太多，这一切有意义，但没那么大。

**Reply by a6749920** [reply]
看你身边朋友多少是买到停工的楼盘了

**Reply by caesarding07** [reply]
可能真的没有雷曼兄弟风险大  这些烂尾楼已经有人买单了 不存在风险

**Comment by C-Joey** [comment]
最多只能验证是否烂尾的情况 其他信息是不可能准确知晓的
谁实名签字谁就会被喝茶 只能在之后的时间里不断修正
起到的作用是有的 能够形成羊群效应 对于真正的困难人群形成一定的安慰效应

**Reply by theCNONE** [reply]
安慰很有限，主要是对银行的震慑吧

**Reply by C-Joey** [reply]
> 安慰很有限，主要是对银行的震慑吧

你都说了对银行震慑 这还不够安慰吗 要知道在这之前 烂尾之后是毫无希望了

**Comment by iosyyy** [comment]
统计这东西并不是为了学术的严谨性 而是为了上面看到能解决问题 至少保证大部分数据证据让他们看到就好了 不然普通人还能怎么办呢 还有别的办法吗
至于你说的
> 在“航空母舰吧”全是兔兔，在“中国房价吧”里面全是神神，在“抗吧”里面全是鼠鼠，真正的因停贷导致的不良资产又占银行资产的多少呢？

和现在的问题完全不一样 重点从来不在于是不是而在于有多少人站出来了能陪看到

**Comment by MrChen2016-08-27** [comment]
无所谓，这肯定不是百分百立马停贷的，如果成功了，等于保护了想主动停贷和被迫停贷的，而这些如果成功了，等于保留了退路对观望的也有利，同时也可以让对方重视这些问题，其实对方也在猜这次会有多大比例，但最后两三年内不复工的话，观望的不也迟早是停贷的。
最好每一个楼都发表一份，立不立马停贷不重要，趁这次大的先发声明掌握保留停贷的权利，如果这个成功了，保留了停贷的退路怎么都不亏不是吗，毕竟拖下去你们也是照样停贷。

**Reply by theCNONE** [reply]
确实，停不停想声明出来，银行也要以所有人都停的后果做打算

**Comment by ethenlong** [comment]
真假自有人去辨别，这里主要还是给那些受害者一个发声的平台。
一个连声音都不让发出来的地方，他们又怎么能呼朋引伴，形成力量呢？

**Comment by cccp057** [comment]
停贷是不是真的不清楚，但是既然到了发声明的地步，有一点是可以明确的，就是这些楼盘烂尾可能性较大。毕竟建设的热火朝天的楼盘你发个烂尾声明，开发商不找你麻烦其他住户也会找你麻烦的

**Reply by ruixiaojia** [reply]
大多都是停工很久了，业主们看不到希望迫不得已才发的声明，如果建设的热火朝天谁会发这些声明来影响自己的楼盘

**Reply by z-huabao** [reply]
不是烂尾可能性很大，是已经烂尾很久的都很多

**Comment by bicycleRiding** [comment]
> 这里面统计的是真正完完全全的断供，还是只是烂尾楼？ 晚上有很多所谓的“烂尾声明”，当然有真的，那是否有假的呢？反正那些“声明”上既没有盖章，也没手印更没签字 我现在发布“重庆保利观澜三期全体业主停贷声明”，是仅仅我停贷，还是真的所有人联合停贷呢？ 所谓的停贷里面又有多少“工贼”呢？ 在“航空母舰吧”全是兔兔，在“中国房价吧”里面全是神神，在“抗吧”里面全是鼠鼠，真正的因停贷导致的不良资产又占银行资产的多少呢？

可以列举哪些是假的证据，然后提个修改的 `pr`

**Comment by Accelerator6666** [comment]
这些公告不管多少是真的多少是假的，至少可以表明了一部分烂尾楼主的态度了。默默忍受不再是唯一选项。

**Comment by paulprogrammer2001** [comment]
单从我所在地区抽样来看，全国烂尾楼数量一定不止200这个数字，甚至还要翻一个数量级；而在全国所有的烂尾楼中，目前有200多处（仓库已列）已知甚至明确已经烂尾，可以被证实为负资产；尽管如你所言，这200处地方的信息不一定真实，可能是一些好事者有意为之，甚至根本不存在停贷（供）行为，但是这些足以说明人们的停贷欲望是强烈了，而我们不可知的就是欲望有多强烈，有多真实；再进一步，以更加严谨的态度审查这些停贷声明和相关材料，至少有十位数的楼盘是确确实实发生了停贷（供）行为。
根据部分楼盘的[停贷数据规模的初步统计](https://stormy-thunbergia-586.notion.site/21dab14200e2478eb91c49b68d16495f)（存在未经核实信息），近些年累积的烂尾楼至少已经影响了40000户家庭。按照每户家庭一对夫妻进行推算，烂尾楼可能会对8万多人的生活甚至生计产生严重影响。统计学难免存在误差和错漏，但他却能很真实的反应一个现象和一个趋势。在我看来，即我国的房地产领域确实存在很大问题，而且很难被纠正，当然这些不是我们可以考虑甚至改变的事情，但是我们有了解真相的权利。试想，如果我们再扩大统计范围，调整统计口径，合理控制误差，让统计数据更加接近于真实。你们觉得结果会是怎么样？或者说你们心中的结果是怎么样？是岁月静好？还是全面加速？
~~这已经不是一个简简单单的业主住房、停贷问题了，而是一定要有人、有机构、有政策为此负责。否则烂尾问题永远没有办法得到改善，恶人永远没有机会得到审判~~

**Comment by zhw173** [comment]
就算有一半是假的，涉及的钱也很多了吧

**Reply by rainforestlin** [reply]
看着绝对数挺大的，但是相对于整个国家的贷款来讲，其实只是很小很小的一部分。

**Comment by 17621014577** [comment]
主要是发声！

**Comment by chenjiasu** [comment]
有假的又能怎样？问题是你们怕什么？逼的别人只能跑到墙外讨论这问题
